nem.pm

```diff
@@ -1096,6 +1096,62 @@ sub ifx_proc {
     &ifx_proc_octet($inst, $res, $prod);
 }
 
+################################################################ qfx_*
+################################################################ Juniper QFX
+
+# use the jnxCosQstatTotalDropPkts counter from JUNIPER-COS-MIB:: as the
+# number for ifOutDiscards
+
+my $qfx_entry = '.1.3.6.1.4.1.2636.3.15.4.1';
+my $qfx_outdiscard = "$qfx_entry.53";
```

Comment:
This is labelled "outdiscard" but the object name has "TotalDropPkts".  Are we just using this terminology to be consistent with the other switches?